bin/xwmeta

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF >&2
+usage: $0 [lint|test]
+EOF
+}
+
+if [[ $# -eq 0 ]]; then
+    usage
+    exit 1
+fi
+
+ACTION="$1"
+shift
+
+cd "$(git rev-parse --show-toplevel)"
+
+for project in libxw; do
+    pushd "$project/"
```

Comment:
`pushd` is like `cd` except that it adds the directory to a stack.. which you can later `popd` from. So
```sh
$ pushd a
$ pushd b
$ pushd c
$ pwd
c

$ popd
$ pwd
b

$ popd
$ pwd
a
```